File: FreeCodeCamp/Javascript/findersKeepers.js
// https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/basic-algorithm-scripting/finders-keepers
// Reprak11

function findElement(arr, func) {
    const found = arr.find(element => func(element) == true);
    return found;
}

console.log(findElement([1, 3, 5, 9], num => num % 2 === 0));

File: FreeCodeCamp/Python/findersKeepers.py
# https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/basic-algorithm-scripting/finders-keepers
# Reprak11

def findElement(arr, func):
    for i in arr:
        if func(i) == True:
            return i
    return


if __name__ == "__main__":
    print(findElement([1, 3, 5, 9], lambda num : num % 2 == 0 ))